<html>
<head>
<title>TestNG:  CalculatorTestTestngSuite</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>CalculatorTestTestngSuite</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>13/3/2</td>
</tr><tr>
<td>Started on:</td><td>Thu Mar 24 21:54:27 IST 2022</td>
</tr>
<tr><td>Total time:</td><td>0 seconds (386 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest.add1Test()'><b>add1Test</b><br>Test class: examplePackageForTestNg.CalculatorTest</td>
<td><div><pre>java.lang.AssertionError: expected [10] but found [111]
	at examplePackageForTestNg.CalculatorTest.add1Test(CalculatorTest.java:13)
... Removed 29 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1816756435", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1816756435'><pre>java.lang.AssertionError: expected [10] but found [111]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at examplePackageForTestNg.CalculatorTest.add1Test(CalculatorTest.java:13)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest@5717c37</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest2.add2Test()'><b>add2Test</b><br>Test class: examplePackageForTestNg.CalculatorTest2</td>
<td><div><pre>java.lang.AssertionError: expected [9] but found [11]
	at examplePackageForTestNg.CalculatorTest2.add2Test(CalculatorTest2.java:20)
... Removed 29 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1200906406", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1200906406'><pre>java.lang.AssertionError: expected [9] but found [11]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at examplePackageForTestNg.CalculatorTest2.add2Test(CalculatorTest2.java:20)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest2@68f4865</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest.add2Test()'><b>add2Test</b><br>Test class: examplePackageForTestNg.CalculatorTest</td>
<td><div><pre>java.lang.AssertionError: expected [9] but found [11]
	at examplePackageForTestNg.CalculatorTest.add2Test(CalculatorTest.java:18)
... Removed 29 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1329315688", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1329315688'><pre>java.lang.AssertionError: expected [9] but found [11]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at examplePackageForTestNg.CalculatorTest.add2Test(CalculatorTest.java:18)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest@5717c37</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest2.add1Test()'><b>add1Test</b><br>Test class: examplePackageForTestNg.CalculatorTest2</td>
<td></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest2@68f4865</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest3.add1Test()'><b>add1Test</b><br>Test class: examplePackageForTestNg.CalculatorTest3</td>
<td></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest3@1a72a540</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest3.add2Test()'><b>add2Test</b><br>Test class: examplePackageForTestNg.CalculatorTest3</td>
<td></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest3@1a72a540</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest2.add3Test()'><b>add3Test</b><br>Test class: examplePackageForTestNg.CalculatorTest2</td>
<td></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest2@68f4865</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest.add3Test()'><b>add3Test</b><br>Test class: examplePackageForTestNg.CalculatorTest</td>
<td></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest@5717c37</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest2.devideByZeroTest()'><b>devideByZeroTest</b><br>Test class: examplePackageForTestNg.CalculatorTest2</td>
<td><div><pre>java.lang.ArithmeticException: / by zero
	at examplePackageForTestNg.Calculator.div1(Calculator.java:21)
	at examplePackageForTestNg.CalculatorTest2.devideByZeroTest(CalculatorTest2.java:53)
... Removed 24 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace24293395", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace24293395'><pre>java.lang.ArithmeticException: / by zero
	at examplePackageForTestNg.Calculator.div1(Calculator.java:21)
	at examplePackageForTestNg.CalculatorTest2.devideByZeroTest(CalculatorTest2.java:53)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest2@68f4865</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest.devideByZeroTest()'><b>devideByZeroTest</b><br>Test class: examplePackageForTestNg.CalculatorTest</td>
<td><div><pre>java.lang.ArithmeticException: / by zero
	at examplePackageForTestNg.Calculator.div1(Calculator.java:21)
	at examplePackageForTestNg.CalculatorTest.devideByZeroTest(CalculatorTest.java:50)
... Removed 24 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace664186542", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace664186542'><pre>java.lang.ArithmeticException: / by zero
	at examplePackageForTestNg.Calculator.div1(Calculator.java:21)
	at examplePackageForTestNg.CalculatorTest.devideByZeroTest(CalculatorTest.java:50)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest@5717c37</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest2.mul1Test()'><b>mul1Test</b><br>Test class: examplePackageForTestNg.CalculatorTest2</td>
<td></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest2@68f4865</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest.mul1Test()'><b>mul1Test</b><br>Test class: examplePackageForTestNg.CalculatorTest</td>
<td></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest@5717c37</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest2.sub1Test()'><b>sub1Test</b><br>Test class: examplePackageForTestNg.CalculatorTest2</td>
<td></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest2@68f4865</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest.sub1Test()'><b>sub1Test</b><br>Test class: examplePackageForTestNg.CalculatorTest</td>
<td></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest@5717c37</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest2.sub2Test()'><b>sub2Test</b><br>Test class: examplePackageForTestNg.CalculatorTest2</td>
<td></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest2@68f4865</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest.sub2Test()'><b>sub2Test</b><br>Test class: examplePackageForTestNg.CalculatorTest</td>
<td></td>
<td>0</td>
<td>examplePackageForTestNg.CalculatorTest@5717c37</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-skipped'>
<tr><td colspan='4' align='center'><b>SKIPPED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest2.add4Test()'><b>add4Test</b><br>Test class: examplePackageForTestNg.CalculatorTest2</td>
<td></td>
<td>0</td>
<td>null</td></tr>
<tr>
<td title='examplePackageForTestNg.CalculatorTest.add4Test()'><b>add4Test</b><br>Test class: examplePackageForTestNg.CalculatorTest</td>
<td></td>
<td>0</td>
<td>null</td></tr>
</table><p>
</body>
</html>